src/VisualStudio/IntegrationTest/New.IntegrationTests/CSharp/CSharpWinForms.cs

```diff
@@ -208,28 +208,32 @@ await TestServices.Workspace.WaitForAllAsyncOperationsAsync(
     [IdeFact]
     public async Task DeleteControl()
     {
-        var project = ProjectName;
-        await TestServices.SolutionExplorer.OpenFileWithDesignerAsync(project, "Form1.cs", HangMitigatingCancellationToken);
-        await TestServices.Editor.AddWinFormButtonAsync("SomeButton", HangMitigatingCancellationToken);
-        await TestServices.SolutionExplorer.SaveFileAsync(project, "Form1.cs", HangMitigatingCancellationToken);
-        await TestServices.SolutionExplorer.SaveFileAsync(project, "Form1.resx", HangMitigatingCancellationToken);
-        await TestServices.Editor.DeleteWinFormButtonAsync("SomeButton", HangMitigatingCancellationToken);
-
-        await TestServices.Workspace.WaitForAllAsyncOperationsAsync(
-            [
-                FeatureAttribute.Workspace,
-                FeatureAttribute.SolutionCrawlerLegacy,
-                FeatureAttribute.DiagnosticService,
-                FeatureAttribute.EditAndContinue,
-                FeatureAttribute.ErrorSquiggles,
-                FeatureAttribute.ErrorList,
-            ],
-            HangMitigatingCancellationToken);
-
-        Assert.Empty(await TestServices.ErrorList.GetBuildErrorsAsync(HangMitigatingCancellationToken));
-        await TestServices.SolutionExplorer.OpenFileAsync(project, "Form1.Designer.cs", HangMitigatingCancellationToken);
-        var actualText = await TestServices.Editor.GetTextAsync(HangMitigatingCancellationToken);
-        Assert.DoesNotContain(@"VisualStudio.Editor.SomeButton.Name = ""SomeButton"";", actualText);
-        Assert.DoesNotContain(@"private System.Windows.Forms.Button SomeButton;", actualText);
+        // Known failure on 32 bit release. Entire phase could eventually be removed. This is a more targeted stopgap until then.
```

Comment:
note: i think we may have test attributes for doing this limiting. @jaredpar ?

Reply:
ah.  [Bitness64] should do it.

Reply:
conditionalidefact flowing from vs-extension-testing. Unblocking dartlab CI until then.